### PR TITLE
[SG-416] Updates to Bitwarden Authenticator

### PR DESCRIPTION
**Pull request by andrebispo5**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Rework on Authenticator UI/UX in order to improve its usability.

## Code changes

Mobile client PR: https://github.com/bitwarden/mobile/pull/1964

- **browser/src/popup/settings/premium.component.ts** 
Changed go back in order to reuse the screen in cipher as well as settings.
- **browser/src/popup/vault/view.component.html** 
- **desktop/src/app/vault/view.component.html**
Added new TOTP link for free users which redirects to purchase premium page.
- **web/src/app/vault/add-edit.component.html**
Added new TOTP link for free users which redirects to purchase premium page.
Removed image of placeholder TOTP and added a faded view with empty TOTP code.

## Screenshots
Desktop:
image
Verification code (TOTP)

Web Vault: 
image
image
image
image
image
image

Browser:
image
image

## Before you submit

```
- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

**Comment by differsthecat** [comment]
This is a huge improvement and looks great! My only feedback is that we use tailwind instead of bootstrap classes on new work if it's not too crazy of a lift

File: apps/web/src/app/vault/add-edit.component.html [hunk 1]
```diff
@@ -179,37 +179,58 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{ title }}</h2>
                 [disabled]="cipher.isDeleted || !cipher.viewPassword || viewOnly"
               />
             </div>
-            <div class="col-6 form-group totp d-flex align-items-end" [ngClass]="{ low: totpLow }">
-              <div *ngIf="!cipher.login.totp || !totpCode">
-                <img
-                  src="../../images/totp-countdown.png"
-                  id="totpImage"
+            <div class="col-6 form-group d-flex align-items-end" [ngClass]="{ low: totpLow }">
```
[the remark this answers is not in the document]
Comment (andrebispo5) [reply apps/web/src/app/vault/add-edit.component.html]: Thanks for the heads up. Converted all to tailwind classes.

**Review by differsthecat — DISMISSED** [review]
Nice work!

File: apps/web/src/app/vault/add-edit.component.html [hunk 2]
```diff
@@ -179,14 +179,41 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{ title }}</h2>
                 [disabled]="cipher.isDeleted || !cipher.viewPassword || viewOnly"
               />
             </div>
-            <div class="col-6 form-group totp d-flex align-items-end" [ngClass]="{ low: totpLow }">
-              <div *ngIf="!cipher.login.totp || !totpCode">
-                <img
-                  src="../../images/totp-countdown.png"
```
Comment (Hinton) [inline apps/web/src/app/vault/add-edit.component.html:185]: Please don't forget to delete this image.

Comment (andrebispo5) [reply apps/web/src/app/vault/add-edit.component.html:185]: done

**Review by differsthecat — APPROVED** [review]
Looks good!

**Pull request merged**